### DISCUSSION TITLE
[A-L] (2023/24) Foglio 3 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#151] [Esercizi - Bertini] [answered]
![Es 3](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/3c679320-5696-4349-8f07-66f1f30fb3d4)

**Comment by Elia-Belli** [comment]
- **(A)** La maestra può realizzare $|\Omega_1|=\binom{30}{4}$ compiti diversi, Carletto riesce a risolvere tutti gli esercizi solo se la maestra sceglie tra quelli che lui ha studiato, ovvero Carletto ha $|A|=\binom{10}{4}$ possibilità che ciò avvenga : $\mathbb{P}(A)= \binom{10}{4}/\binom{30}{4}$
- **(B)** Con questo metodo la maestra può realizzare $|\Omega_2|= \binom{50}{5}\binom{30}{4}\binom{10}{1}$ compiti diversi.
- **(C)** Carletto riesce a risolvere tutti gli esercizi solo se la maestra sceglie tra quelli che lui ha studiato: $|C|=\binom{20}{5}\binom{10}{4}\binom{5}{1}$, $\mathbb{P}(C)= |C|/|\Omega_2|$
- **(D)** Siccome Carletto svolge l'esercizio solo se lo ha imparato a memoria, dobbiamo imporre che la maestra ne scelga rispettivamente 3, 2, 1 tra quelli memorizzati e i restanti a caso. Quindi le facciamo prima scegliere tra quelli memorizzati, e poi i restanti a caso, per ogni materia: $|D|= \binom{20}{3}\binom{10}{2}\binom{5}{1}\binom{47}{2}\binom{28}{2}$
$\mathbb{P}(D)= |D|/|\Omega_2|$

**Comment by atyion** [comment]
Riguardo la parte in cui calcoli |D|:
Non sarebbe corretto dire che bisogna scegliere 2 esercizi a caso delle equazioni su 47 e 2 esercizi a caso di geometria su 28?

**Comment by CiottoloMaggico** [comment, marked as answer]
# Soluzione confermata in classe
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/79638284/f59efbf5-57a3-4f71-ae98-db294f6890a3)
@Elia-Belli non concordo con l'ultimo punto, scegliendo le domande rimanenti nel tuo modo stai contando i casi in cui carletto fa **almeno** 3, 2, 1 domande esatte, seguendo la traccia ne deve fare precisamente 3,2,1.

**Comment by Elia-Belli** [comment]
@CiottoloMaggico si, io l ho inteso come un almeno. Però rileggendo forse è più corretto un "esattamente".

**Comment by CiottoloMaggico** [comment]
Allora è giusto se si intende come almeno
